### DISCUSSION TITLE
Javascript for browsers vs Javascript for Node

### DISCUSSION THREAD

**Discussion by akkshayTandon** [codethesaurus/codethesaur.us#656] [General]
I don't know why i am asking , but have this question tinkling in my mind.
Should Javascript be broken into JS for browsers ans JS for Node ? Since documentation does mentions that different breeds/version of same language could exist .
(Didn't mentioned in the issue since a discussion first seemed reasonable)

**Comment by grady2002** [comment]
idk the context but here is my opinion :D
well, the language shouldn't be broken down, like people are acquainted with one JS and learning another version of the same language can be frustrating for some people. so far its going good !!!

**Comment by geekygirlsarah** [comment]
I think for the most part they're the same. I asked a friend about it and he couldn't really see much of a need to differentiate between browser and non-browser (Node) JS. Maybe for parts where it differs it could have comments. (Like Node has functions for reading the file system that you can't do in-browser.) If it differed drastically (like foundational/primitive stuff) I could uses a use case for this but presently don't.

I'm open to ideas though!

**Reply by grady2002** [reply]
Well said @geekygirlsarah !!
